grr-ui/src/templates/BackgroundImageList/BackgroundImageList.js

```diff
@@ -0,0 +1,38 @@
+import React, {Component} from 'react';
+import './BackgroundImageList.css';
+import {Menu, MenuItem, MenuDivider, Switch} from '@blueprintjs/core';
+
+
+class BackgroundImageList extends Component {
+
+    constructor(){
+        super();
+        this.handleClick = this.handleClick.bind(this);
+    }
+
+    handleClick(e){
+        var selectedImage = e.target.textContent;
+        this.props.onSelectedBackground(selectedImage);
+    }
+
+
+    render() {
+        return (<div className="list-container">
+            <Menu className="pt-large">
+                <li className="pt-menu-header centered-header">
+                    <h6>Default Background Options</h6>
+                </li>
+                <MenuItem text="stock360.png" onClick={this.handleClick}/>
```

Comment:
The list of MenuItem and MenuDivider's is pretty repetitive. Could the list of images be in an array in a configuration file? Then import the config and image array. Loop the array and add in the MenuItems.

```javascript
return (
   <div>
    {this.props.level.map((item, index) => (
       <span className='indent' key={index} />
    ))}
    "Some text value"
   </div>
);
```

Reply:
Fixed here: b7aeab8